docs/modules/ROOT/pages/glossary/index.adoc

```diff
@@ -101,5 +98,8 @@ A mechanism to secure the software supply chain by recording events in a user-de
 **Tekton integration testing** +
 A process that uses Tekton tasks to support the setup and execution of dynamic application tests against container images.
 
+**Tekton task** +
+One or more steps that run container images. Each container image performs a piece of construction work.
+
 **workspace** +
```

Comment:
Are you planning to add a definition for a Tekton workspace as well?

Reply:
yes. i'm keeping a list in the related Jira story (link in this PR's descr) of terms we need to define.